apps/docs/src/app/core/component-docs/popover/popover-docs.module.ts

```diff
@@ -32,6 +32,7 @@ import { BarModule } from '@fundamental-ngx/core/bar';
 import { SegmentedButtonModule } from '@fundamental-ngx/core/segmented-button';
 import { FormModule } from '@fundamental-ngx/core/form';
 import { CheckboxModule } from '@fundamental-ngx/core/checkbox';
+import { LinkModule } from '@fundamental-ngx/core';
```

Comment:
I don't think this is the correct way to import modules. Please use the new approach